Assets/Scripts/ShortcutTrigger.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShortcutTrigger : MonoBehaviour
+{
+    public bool visited = false;
```

Comment:
do you need it have it Serialized? It doesn't change anything apart from showing it in script component in for an object ofc, but just for a good practice 🤔

Reply:
it nice to see it change in the editor for debuging and testing